tests/soak-tests/pom.xml

```diff
@@ -63,8 +116,20 @@
          <type>test-jar</type>
       </dependency>
       <dependency>
-         <groupId>junit</groupId>
-         <artifactId>junit</artifactId>
+         <groupId>org.apache.activemq</groupId>
+         <artifactId>activemq-client</artifactId>
+         <version>${activemq5-version}</version>
```

Comment:
This is a managed dependency, should be able to drop the version element. (The exclusions are managed also, but can be nice for clarity)